source/views/sis/student-work/detail.js

```diff
@@ -0,0 +1,206 @@
+          detail={trimmedOffice}
+          accessory="DisclosureIndicator"
+          onPress={() => openEmail(email, title)}
+        />
+      </Section>
+    : null
+}
+
+function renderHours(timeOfHours: string, hoursPerWeek: string) {
+  const time = fastGetTrimmedText(timeOfHours)
+  const hours = fastGetTrimmedText(hoursPerWeek)
+  const ending = hours == 'Full-time' ? '' : ' hrs/week'
+  return time && hours
+    ? <Section header="HOURS">
+        <Cell
+          cellStyle="Subtitle"
+          title={time}
+          detail={hoursPerWeek + ending}
+        />
+      </Section>
+    : null
+}
+
+function renderDescription(description: string) {
+  const trimmedDescription = description.replace(/\t/g, ' ')
+  return trimmedDescription
+    ? <Section header="DESCRIPTION">
+        <Cell
+          cellContentView={
+            <Text selectable={true} style={styles.selectable}>
+              {trimmedDescription}
+            </Text>
+          }
+        />
+      </Section>
+    : null
+}
+
+function renderSkills(skills: string) {
+  const trimmedSkills = skills.replace(/\t/g, ' ')
+  return trimmedSkills
+    ? <Section header="SKILLS">
+        <Cell
+          cellContentView={
+            <Text selectable={true} style={styles.selectable}>
+              {trimmedSkills}
+            </Text>
+          }
+        />
+      </Section>
+    : null
+}
+
+function renderComments(comments: string) {
+  const trimmedComments = comments.replace(/\t/g, ' ')
+  return trimmedComments
+    ? <Section header="COMMENTS">
+        <Cell
+          cellContentView={
+            <Text selectable={true} style={styles.selectable}>
+              {trimmedComments}
+            </Text>
+          }
+        />
+      </Section>
+    : null
+}
+
+function renderLinks(links: string[]) {
+  return links.length
+    ? <Section header="LINKS">
+        {links.map((url, i) => (
+          <Cell
+            key={i}
+            cellStyle="Title"
+            title={url}
+            accessory="DisclosureIndicator"
+            onPress={() =>
+              openUrl(/^https?:\/\//.test(url) ? url : `http://${url}`)}
+          />
+        ))}
+      </Section>
+    : null
+}
+
+function fixupEmailFormat(email: string) {
+  if (!/@/.test(email)) {
+    // No @ in address ... e.g. smith
+    return `${email}@stolaf.edu`
+  } else if (/@$/.test(email)) {
+    // @ at end ... e.g. smith@
+    return `${email}stolaf.edu`
+  } else {
+    // Defined address ... e.g. smith@stolaf.edu
+    return email
+  }
+}
+
+function openEmail(email: string, subject: string) {
+  let address = fixupEmailFormat(email)
+  Communications.email([address], null, null, subject, '')
+}
+
+function parseLinks(data: string) {
+  const allLinks = data.split(' ')
+  if (!allLinks.length) {
+    return []
+  }
+  return allLinks.filter(w => /^https?:\/\//.test(w))
+}
+
+export default function JobDetailView({job}: {job: JobType}) {
+  const title = job.title.trim() || ''
```

Comment:
Please remove the `|| ''` from each of these that call `.trim()` – as it stands, you'll either crash (because there's no string to trim) or you'll get back a string (because you called trim) but you'll never use the right-hand value.